src/os/README.md

```diff
@@ -189,3 +189,38 @@ final class posix_spawn_file_actions_adddup2 implements PosixSpawnFileActionsSet
 
 type posix_spawn_file_actions_t = vec<PosixSpawnFileActionsSetter>;
 ```
+
+#### Special encoding for unordered setters as a `shape`
+
+For some C data types the setters are orthogonal to each other, and the order to
```

Comment:
This should swap with the above, or entirely remove 'default' wording for either case.

Reply:
Order swapped